src/custom/components/WalletModal/WalletModalMod.tsx

```diff
@@ -325,6 +326,11 @@ export default function WalletModal({
           <Option
             id={`connect-${key}`}
             onClick={() => {
+              // Solves the problem of opening both Coinbase and Metamask popups
+              if (option.name === 'MetaMask') {
```

Comment:
what about other options? If we go this route, should we have a "list of injected wallets", basically with the values of that parameter?

Reply:
Also, why do we need to add this? is not `tryActivation` enough?